packages/server/src/backup/backup-manual-test.ts

```diff
@@ -0,0 +1,37 @@
+import { BackupService } from './backup.service';
+
+/*
+ * Manual tests for the backup service that bypass the cron jobs.
+ *
+ * Command: yarn tsx backup-manual-test.ts
+ *
+ * Note: when using tsx, the relative paths are resolved from the root of this node project (packages/server).
+ */
+
+async function runManualTests() {
```

Comment:
wait why is this here? You should be able to call the backup services and bypass the cron jobs just fine?

Reply:
Tsx was added so that we could run the backup-manual-test.ts without transpiling it to JS. I wrote this file up mainly because it just makes manually testing the service a one-line process. I feel that in the future, there might be a use case where we might want to ensure that the backup services are still working in deployment or want to bypass cron in one line. If you really feel that this addition is unnecessary, I can remove it. I just think that it does no harm to have it just as an extra tool on our belt for interacting with the backup service.

Reply:
Ah I see, i guess the use case for wanting to manually run backups makes sense. 

Though just a question, when we run these manual tests, does it actually save backups? Since it may be wise to rename to not confuse it with our actual tests, maybe just call it runManualBackups and name the file backup-manual.ts kinda thing

Reply:
That makes sense. I'll change the filename in a second.

Reply:
I changed the wording on this filename and the code inside so it should be clearer now. I also updated the BACKUPS.md to reflect the new backup cycle for uploads. Everything should be good now.